addon/instance-initializers/link-handler.js

```diff
@@ -46,6 +46,10 @@ export function shouldHandleLink(node, base = location) {
   } else if (node.getAttribute('data-ember-action')) {
     // ignore clicks from ember actions
     return false;
+  }  else if (node.getAttribute('data-skip-link-handler')) 
+    // ignore previous handling methods and opens as a normal link
+  {
```

Comment:
Looks good but can you move this bracket to same line as the if else statement to match up with the rest of the code here?